### ISSUE TITLE
[FEATURE] Add the function of hiring new staff

### ISSUE THREAD

**Issue by JackYu-QQQ123** [COMP3122-2324-Project/Simpsyber#38]
As the company's products become more complex, we need more specialized staff to complete the work. For example, we have new features for every four weeks of levels completed. Allow the player, as project manager, to select new employees for the team based on price and skill set. The salaries and skill sets of the employees vary greatly depending on their level of specialization. Players must choose the right employee based on their knowledge of the game; otherwise, choosing the wrong employee can add to heavy costs.

**Comment by Alexachenggg** [comment]
Totally agree with you! I think having more staffs will make the game more interesting and smoother to play. 

Moreover, there is a "+" displayed on the blank photo next to the staff photo in the "select staff" function page:
<img width="709" alt="螢幕截圖 2024-03-03 下午11 20 52" src="https://github.com/COMP3122-2324-Project/Simpsyber/assets/156781919/c873e4a4-ff47-45e3-b3f0-fa6e99d73b78">

which misled the user into thinking they could add staff or recruit staff. Therefore, the place could be the vacancy for adding new staff in the game and also make the “+” clickable for user to do the operation/transaction.

**Comment by CasparFHM** [comment]
After having the hiring of a new staff function, I think a function that can fire a staff should be added. 
As more staff is employed, it is harder for the player to manage different staff.
Also, the player has to pay salaries for the staff after a period of time, and it may cause a waste of money if the staff is not suitable. Therefore, the function that can fire a staff should also be added.
![7](https://github.com/COMP3122-2324-Project/Simpsyber/assets/156775084/8858d197-424d-4329-93b6-455682cebe77)

**Comment by CHEUNG-Choi-Chun-Ifan-23041078d** [comment]
I agree with you, I also suggest that the game can let the staff enhance their skill in different fields. It can make the staff become more efficient and it can reduce the time on the whole process and it may help the company's work to make a better app for the users.

**Comment by YANGZhaochen2002** [comment]
I agree with your point of view. Even though some employees have been consistently performing well, by the time we reach the QA phase, there are only two efficient employees left. Moreover, regarding the entire development stage, it is necessary to adjust the proportion of employees with different specializations. Therefore, I believe allowing players to control the ratio of different employees within the company is essential.

**Comment by bronyazzy** [comment]
I agree with the idea of adding new employees, however, I think that when the number of employees increases there should be an increase in debuffs brought about by management difficulties, such as a decrease in efficiency per employee

**Comment by lordsavor** [comment]
I agree with this! I also added another feature relating to your feature. WIth both of these feature, it could improve the game by adding more depth and strategy with the fund system.
